src/main/clojure/conexp/fca/factorization.clj

```diff
@@ -0,0 +1,927 @@
+(ns conexp.fca.factorization
+    (:require [conexp.base :refer :all]
+              [conexp.fca.fast :refer [to-binary-matrix]]
+              [conexp.fca.contexts :refer :all]))
+
+(defprotocol factorization-protocol
+  (context [this] "Returns the context that is generated after using the algorithm algo k-times on it")
+  (object-factor [this] "Returns the object-factor of the context generated by the algorithm")
+  (attribute-factor [this] "Returns the attribute-factor of the context generated by the algorithm"))
+
+(defrecord factorization-record [context object-factor attribute-factor]
+  factorization-protocol
+  (context [this] context)
+  (object-factor [this] object-factor)
+  (attribute-factor [this] attribute-factor))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; helper functions
+
+(defn make-matrix-from-concept
+  [a b n m]
+  ;; make matrix from concept a&b with n m 
+  (loop [i 0 j 0 matrix []]
+    (if (> (inc j) n)
+      matrix
+      (recur
+       (cond (>= (inc i) m) 0 :else (inc i))
+       (cond (>= (inc i) m) (inc j) :else j)
+       (cond (and
+              (some #(= j %) a)
+              (some #(= i %) b))
+             (conj matrix 1) :else (conj matrix 0))))))
+
+(defn transpose
+  "Transposes matrix"
+  [a]
+  (into [] (apply map vector a)))
+
+(defn make-matrix-from-context
```

Comment:
Why have a string to 1/0 conversion here? Seems like a function from conexp.io.contexts should have been used

Reply:
Haven't found any context -> 1/0-matrix conversion